core/src/components/buttons/buttons.ios.scss

```diff
@@ -99,7 +99,7 @@
   @include padding(0);
   @include margin(0);
 
-  font-size: 28px;
+  font-size: 1.65em;
```

Comment:
This value is calculated off of `17px` which is the default `font-size` here, not `16px`